geoips/commandline/list_available_plugins.py

```diff
@@ -0,0 +1,74 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Simple script to list available plugins for each interface."""
+import warnings
+from geoips.interfaces.base import BaseInterface
+from geoips import interfaces
+from geoips.commandline.log_setup import setup_logging
+from geoips.commandline.args import get_argparser, check_command_line_args
+
+# Always actually raise DeprecationWarnings
+# Note this SO answer https://stackoverflow.com/a/20960427
+warnings.simplefilter("always", DeprecationWarning)
+
+
+def main():
+    """Script to list all modules available in the current geoips instantiation."""
+    supported_args = ["logging_level"]
+
+    # Use standard GeoIPS argparser - use standard formatting for standard args.
+    parser = get_argparser(
+        arglist=supported_args,
+        description="List available plugins within the current geoips instantiation.",
+    )
+    # Add our specific "interfaces" argument.
+    parser.add_argument(
+        "interfaces",
+        nargs="*",
+        default=None,
+        help="""List of interfaces to include.  If None, include all interfaces.""",
+    )
```

Comment:
@jsolbrig Here I used the default "geoips" argparser, then added a "custom" argument.

Reply:
Thanks, this works well for my use case. It looks like you added my extra argument to the CLI, too!